app/controllers/api/v1x0/actions_controller.rb

```diff
@@ -33,6 +34,19 @@ def create
       def action_params
         params.permit(:operation, :processed_by, :comments)
       end
+
+      def rbac_scope(relation)
+        access_obj = RBAC::Access.new('actions', 'read').process
```

Comment:
with this round of refactoring, now we can extract the first three lines into one method that can be shared by all controllers. This method returns `access_obj` if we need to further narrow the scope. Return `nil` if it is admin.